client/src/app/shared/models/constants.ts

```diff
@@ -491,7 +491,7 @@ export class DeploymentCenterConstants {
   public static readonly editReleaseDefinitionPermission = 2;
 
   // Agent queues
-  public static readonly agentQueueNames = ['Hosted VS2017', 'Hosted Ubuntu 1604', 'Default'];
+  public static readonly agentQueueNames = ['Hosted VS2017', 'Default'];
```

Comment:
Why have we removed it? WE anyway check that any one of them you have the right?

Reply:
User might have permission to only use Hosted Ubuntu and hence this check will pass, but ultimately CD job will fail as it will check for permission on Hosted VS2017 (and on Default as a fallback).
All Deployment center scenarios only require VS2017 and hence we thought about removing ubunutu.